backend/src/user/user.module.ts

```diff
@@ -1,9 +1,10 @@
 import { Module } from '@nestjs/common';
 import { UserController } from './user.controller';
 import { UserService } from './user.service';
+import { PrismaModule } from '../prisma/prisma.module';
 
 @Module({
-  imports: [],
+  imports: [PrismaModule],
```

Comment:
instead of adding Prisma module to each module. You can direcly add it to AppModule and it would work.